### PR TITLE
feat(Embedding): added from_embeddings_to_instructions() and score_instructions()

### PR DESCRIPTION
**Pull request by itchingpixels**

### CODE REVIEW

File: embedding/instructions.py [hunk 1]
```diff
@@ -36,14 +37,41 @@ def __embed_instruction(
...
+
+def from_embeddings_to_instructions(
+    embeddings: torch.Tensor, dataconfig: DataConfig
+) -> list[list[Instruction]]:
+    assert embeddings.shape[1] % 3 == 0
+    no_of_instructions = int(embeddings.shape[1] / 3)
+    embeddings = embeddings.cpu().detach().numpy()
+
+    def single_embedding_to_instruction(embedding: np.ndarray) -> Instruction:
+        if embedding[0] < 0.5:
```
Comment (szemyd) [inline embedding/instructions.py:61]: Do I understand this correctly? You are taking <0.5 as being a circle and >0.5 being a translate function? Would this be extensible to more operations?

File: utils/types.py [hunk 2]
```diff
@@ -7,12 +7,18 @@
 class Circle:
     r: float
 
+    def get_parameters(self) -> list[float]:
```
Comment (szemyd) [inline utils/types.py:10]: We'll need to merge this with mine.

**Review by szemyd — APPROVED** [review]

File: embedding/instructions.py (hunk 1 above)
Comment (itchingpixels) [reply embedding/instructions.py:61]: soon we'll have something more scalable :)

File: utils/types.py (hunk 2 above)
Comment (itchingpixels) [reply utils/types.py:10]: makes sense!

**Pull request merged**